### PR TITLE
Feedback inquiry to users in 3rd week, upgrade proposal in 7th week

### PR DESCRIPTION
**Pull request by patrickvenetz**
This Pull Request adds two new scheduled transactional emails:

1. Send **feedback inquiry** via email to users which are in their 3rd week (ever).
2. Send **upgrade proposal** via to users which are in 7th week of their MONTHLY_ABO membership.

These are implemented as two dedicated scheduler "jobs". Each run a SQL statement query, then sending emails. Jobs should run every 10 minutes.

Content in transactional emails:
- [Sind Sie zufrieden?](https://publikator.republik.ch/repo/republik/newsletter-editorial-sind-sie-zufrieden/tree) (feedback)
- [Wechseln Sie zur Jahresmitgliedschaft!](https://publikator.republik.ch/repo/republik/newsletter-editorial-wechseln-sie-zur-jahresmitgliedschaft/tree) (upgrade)

**Migrations**

a) Before deployment, prepare `mailLog` records to avoid sending transactionals to those who received campaign mail already:

    cat ~/Desktop/membership_owner_feedback.csv | cut -d, -f1 | OVERWRITE_ENV=./.republik-api.env packages/mail/script/importMailLog.js --type membership_owner_feedback --keys null --now 2020-05-12T11:30+0200 --no-dry-run

    cat ~/Desktop/membership_owner_upgrade_monthly.csv | cut -d, -f1 | OVERWRITE_ENV=./.republik-api.env packages/mail/script/importMailLogUpgradeMonthyl.js --type membership_owner_upgrade_monthly --now 2020-07-07T12:00+0200 --no-dry-run

Checklist:
- [x] Reinstate membership counter
- [x] Block recipients which were already targeted via campaign

### CODE REVIEW

**Comment by patrickvenetz** [comment]
@patte If you were to be so kind and have a sneak peak into this. Assuming this is to your liking, another transactional email will come along the same way.

This is ought to send a feedback inquiry via email to users which are in their 3rd week (ever).

It is intentionally not part of `servers/republik/modules/crowdfundings/lib/scheduler/owners.js` but in same folder, as it …
- … allows a more straight forward approach
- … is it somewhat self contained
- … won't require bloating "owners" scheduler any further

File: servers/republik/modules/crowdfundings/lib/scheduler/feedback.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      GROUP BY u.id
+    )
+    
+    SELECT *
+    FROM "eligables" e
+    JOIN "users" u ON u.id = e."userId"
+
+    -- Scope to days behind a user between 21 and 27 days (3rd week)
+    WHERE e."daysBehind" BETWEEN 21 AND 27
+  `)
```
Comment (patte) [inline servers/republik/modules/crowdfundings/lib/scheduler/feedback.js:46]: This query fails for short periods in the past. Let's say for example a user had a (monthly) abo which started "2020-01-02 12:00:00" and ended "2020-01-26 12:00:00" (due to cancelling it at that day immediately). The query above would match it with days behind 24 even though the membership endend a long time ago.

Comment (patrickvenetz) [reply servers/republik/modules/crowdfundings/lib/scheduler/feedback.js:46]: This is hindered a little earlier, basically addressing scenarios (a) in which a membership is no longer active but falls into 21-27 day period or (b) a membership which has been cancelled.

```sql
      -- A membership which is …
      JOIN "memberships" ma ON ma."userId" = u.id
        -- … still active
        AND ma.active = TRUE
        -- … not cancelled
        AND ma.renew = TRUE
```

File: servers/republik/lib/MembershipStats/evolution.js [hunk 2]
```diff
@@ -274,8 +274,15 @@ const sumBucketProps = async (
...
+  context,
+  moment().format('YYYY-MM'),
+  { add: ['active', 'overdue'] }
+)
+
 module.exports = {
   createCache,
   populate,
-  sumBucketProps
+  sumBucketProps,
```
Comment (patte) [inline servers/republik/lib/MembershipStats/evolution.js:286]:
```suggestion
```

File: servers/republik/modules/crowdfundings/lib/scheduler/upgrade.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      JOIN "membershipPeriods" mp ON mp."membershipId" = m.id
+
+      GROUP BY u.id, m.id
+    )
+
+    SELECT *
+    FROM "eligables" e
+    JOIN "users" u ON u.id = e."userId"
+
+    -- Scope to days behind in membership between 45 and 49 days (7th week)
```
Comment (patte) [inline servers/republik/modules/crowdfundings/lib/scheduler/upgrade.js:55]:
```suggestion
    -- Scope to days behind in membership between 42 and 49 days (7th week)
```

**Pull request merged**